src/schema/query/pullJobs.query.ts

```diff
@@ -18,6 +19,15 @@ export default {
     afterCursor: { type: GraphQLID },
   },
   async resolve(parent, args, context) {
+    const first = args.first || DEFAULT_FIRST;
```

Comment:
@ketanreliefapps 
you can put that in a shared method, it's a lot of duplicated code

Reply:
Done